primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -273,11 +273,20 @@
             return dayIndex >= 7 ? dayIndex - 7 : dayIndex;
         },
 
+        getFirstDayOfWeek: function () {
+            return this.options.locale.firstDay !== undefined ? this.options.locale.firstDay : this.options.locale.firstDayOfWeek;
```

Comment:
This won't work?
```suggestion
            return this.options.locale.firstDay || this.options.locale.firstDayOfWeek;
```

Reply:
no because if first day is `0` that is JS falsy so it will then go to first dayofWeek right it needs to check for undefined not just for falsiness?  And that is how it already was in the code I just moved it to a function.

Reply:
Ah! Alright. Need to keep that in mind.